docs/quality-principles/introduction.md

```diff
@@ -0,0 +1,44 @@
+# Dasharo Quality Principles
+
+## Secure Delivery
+
+Each Dasharo release is secured cryptographically, ensuring with RSA 4096 keys,
+authenticity of binary SHA-256 integrity checksum. The three-step key
+verification assuming that your client receives a clean file without any
+third-party modifications. Integrity and signature verification procedure is
+included so that the key compliance can be easily checked.
+
+## Secure Maintenance
+
+Dasharo doesn't believe in security through obscurity. That is why each Dasharo
+firmware is supported with the system of Transparent Validation, proving its
+efficiency and quality with testing results. As an Open Source product, it
+provides fast emergency releases and patches together with long-lasting
+maintenance.
+
+## Credibility
+
+Dasharo values credibility. Each release is extended by a validation report
+based on a project charter and a complete changelog with a description of the
+product versioning.
+
+## Transparency
+
+Generic testing procedures for a given Dasharo product line are included in the
+product documentation. Specific testing procedures are delivered in the form of
+a PDF document with a release.
+
+## Availability
+
+Dasharo product line page includes hardware compatibility platform list. The
+platform for which the new product line is created, becomes it's reference
```

Comment:
platform for which the new product line is created **-** becomes **its'** reference